### PR TITLE
Add ability to change volume using the mouse wheel

### PR DESCRIPTION
**Pull request by mgoulao**
The delay can be a problem when the user scrolls faster

### CODE REVIEW

File: Xpotify/Controls/NowPlayingView.xaml [hunk 1]
```diff
@@ -493,6 +493,7 @@
                                                                         IsThumbToolTipEnabled="True"
                                                                         Value2="{x:Bind ViewModel.Volume, Mode=TwoWay}"
                                                                         Maximum="100"
+                                                                        PointerWheelChanged="SliderExtended_PointerWheelChanged"
```
Comment (MahdiGhiasi) [inline Xpotify/Controls/NowPlayingView.xaml]: Please rename this event to `VolumeSlider_PointerWheelChanged` to match the other event of this element.

File: Xpotify/Scripts/Common/mouseWheelListener.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    function volumeMouseWheelHandler(e) {
+        var { deltaY } = e;
+        var currentVolume = StatusReport.getVolume();
+        var newVolume = Math.max(Math.min(currentVolume - (deltaY / 1000), 1), 0);
+
+        Action.seekVolume(newVolume);
+    }
+
+    function setVolumeBarListener() {
+        var volumeBar = document.querySelector("#main > div > div.Root__top-container > div.Root__now-playing-bar > footer > div.now-playing-bar > div.now-playing-bar__right > div > div > div.volume-bar > div");
```
Comment (MahdiGhiasi) [inline Xpotify/Scripts/Common/mouseWheelListener.ts]: This selector is too strict, and so it's prone to breaking by changes in PWA. Please use a more loose selector such as `.Root__top-container .Root__now-playing-bar .now-playing-bar__right__inner .volume-bar > div` for this.

File: Xpotify/Scripts/Common/mouseWheelListener.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿/// <reference path="statusReport.ts" />
+/// <reference path="action.ts" />
+
+namespace XpotifyScript.Common.MouseWheelListener {
+
+    declare var Xpotify: any;
```
Comment (MahdiGhiasi) [inline Xpotify/Scripts/Common/mouseWheelListener.ts]: This variable declaration seems to be unnecessary on this file.

**Review by MahdiGhiasi — CHANGES_REQUESTED** [review]
Sorry for the late response, and thanks for the pull request! 😃

I've left a few comments here, please apply and push the necessary minor changes. I will then merge it into master.

Also about the issue when the user scrolls faster in now playing page, it seems to me that the real volume change functionality is working fine, but the UI doesn't update until a second after. This will likely be solved automatically when it's merged to master, as I have a change where it forces the UI to update when `LastPlayStatus.Changed` is invoked, so don't worry about it.

**Comment by mgoulao** [comment]
All changes requested added

**Comment by MahdiGhiasi** [comment]
Thanks. I will merge this with master.

**Pull request merged**